src/about.html

```diff
@@ -206,89 +206,89 @@
               <div class="flex items-center gap-2 sm:gap-2.5 2xl:gap-3.5">
-                <div class="border-star p-3.5 sm:p-4 2xl:p-5 rounded-full">
+                <div class="border border-[#703BF7] p-3.5 sm:p-4 2xl:p-5 rounded-full">
                   <img
                     src="/public/assets/aboutpage/icon-excellent.png"
                     alt="icon-excellent"
                   />
                 </div>
-                <div class="font 2xl:text-2xl sm:text-xl text-lg text-color">
+                <div class="font 2xl:text-2xl sm:text-xl text-lg text-[#FFFFFF]">
                   <p>Excellence</p>
                 </div>
               </div>
               <div
-                class="font-medium text-sm sm:text-base 2xl:text-lg text-color-addition-info"
+                class="font-medium text-sm sm:text-base 2xl:text-lg text-[#999999]"
               >
                 We set the bar high for ourselves. From the properties we list
                 to the services we provide.
               </div>
             </div>
           </div>
           <!-- border -->
-          <div class="border-bottom"></div>
+          <div class="border border-[#262626]"></div>
           <!-- 2 rows in this div -->
           <div
             class="grid grid-flow-row sm:grid-flow-col justify-between gap-5 sm:gap-6 2xl:gap-8"
           >
             <!-- box one -->
             <div class="flex flex-col gap-2 sm:gap-4 2xl:gap-5">
               <div class="flex items-center gap-2 sm:gap-2.5 2xl:gap-3.5">
-                <div class="border-star p-3.5 sm:p-4 2xl:p-5 rounded-full">
+                <div class="border border-[#703BF7] p-3.5 sm:p-4 2xl:p-5 rounded-full">
                   <img
                     src="/public/assets/aboutpage/clients.png"
                     alt="clients image"
                   />
                 </div>
-                <div class="font 2xl:text-2xl sm:text-xl text-lg text-color">
+                <div class="font 2xl:text-2xl sm:text-xl text-lg text-[#FFFFFF]">
                   <p>Client-Centric</p>
                 </div>
               </div>
               <div
-                class="font-medium text-sm sm:text-base 2xl:text-lg text-color-addition-info"
+                class="font-medium text-sm sm:text-base 2xl:text-lg text-[#999999]"
               >
                 Your dreams and needs are at the center of our universe. We
                 listen, understand.
               </div>
             </div>
             <!-- border -->
-            <div class="border-button"></div>
+            <div class="border border-[#262626]"></div>
             <!-- second box -->
             <div class="flex flex-col gap-2 sm:gap-4 2xl:gap-5">
               <div class="flex items-center gap-2 sm:gap-2.5 2xl:gap-3.5">
-                <div class="border-star p-3.5 sm:p-4 2xl:p-5 rounded-full">
+                <div class="border border-[#703BF7] p-3.5 sm:p-4 2xl:p-5 rounded-full">
```

Comment:
this border color is used at many places as well

Reply:
just use in about page i think not need to change it